curriculum/challenges/english/25-front-end-development/lab-lunch-picker-program/66db529d37ad966480ebb633.md

```diff
@@ -41,7 +41,7 @@ assert.isArray(adultsMeal);
 Your `kidsMeal` array should contain atleast three items.
```

Comment:
```suggestion
Your `kidsMeal` array should contain at least three items.
```

Reply:
@zairahira it looks like we still have this last suggested change